libs/model/src/aggregates/user/signIn/utils.ts

```diff
@@ -25,10 +25,10 @@ export function constructFindAddressBySsoQueryFilter(
     query += ` oauth_username = :oauthUsername`;
   } else if (['google', 'email', 'apple'].includes(ssoInfo.provider)) {
     query += `oauth_email = :oauthEmail`;
-  } else if (['phone_number'].includes(ssoInfo.provider)) {
+  } else if (['phone_number', 'SMS', 'phone'].includes(ssoInfo.provider)) {
```

Comment:
`ssoInfo.provider === 'SMS'` -> `SMS` is the only one that exists in the `ssoInfo.provider` type